tests/e2e/specs/shopper/checkout-shipping.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { merchant } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNormalPagePermalink,
+	shopper,
+	visitPostOfType,
+} from '../../../utils';
+
+const block = {
+	name: 'Checkout',
+};
+
+const simpleProductName = '128GB USB Stick';
+const freeShippingName = 'Free Shipping';
+const freeShippingPrise = '$0.00';
+const normalShippingName = 'Normal Shipping';
+const normalShippingPrice = '$20.00';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( `${ block.name } Block (frontend)`, () => {
+	let productPermalink;
+
+	beforeAll( async () => {
+		// prevent CartCheckoutCompatibilityNotice from appearing
+		await page.evaluate( () => {
```

Comment:
I wonder if we can more this to a util since it's used in many of the e2e tests now.

Reply:
Thanks for pointing this out. I created the file `tests/utils/merchant.js` and added the following two functions:

```js
preventCompatibilityNotice: async () => {
	await page.evaluate( () => {
		localStorage.setItem(
			'wc-blocks_dismissed_compatibility_notices',
			'["checkout"]'
		);
	} );
},

reactivateCompatibilityNotice: async () => {
	await page.evaluate( () => {
		localStorage.removeItem(
			'wc-blocks_dismissed_compatibility_notices'
		);
	} );
},
```

This way, I can call them in `beforeAll()` and `afterAll()` as `await merchant.preventCompatibilityNotice();` and `await merchant.reactivateCompatibilityNotice();`. Is that what you had in mind?